shell/middleware/authenticated.js

```diff
@@ -91,11 +90,6 @@ export default async function({
     // Load settings, which will either be just the public ones if not logged in, or all if you are
     await fetchInitialSettings(store);
 
-    // Set the favicon - use custom one from store if set
-    if (!haveSetFavIcon()) {
-      setFavIcon(store);
-    }
```

Comment:
The brand mixin is used everywhere that the authenticated middleware is used so this should be fine.